examples/.i18nrc.json

```diff
@@ -0,0 +1,12 @@
+{
+  "paths": {
+    "embeddableExamples": "embeddable_examples",
+    "expressionsExample": "expressions_example",
+    "multipleDataSourceExample": "multiple_data_source_examples",
+    "searchExamples": "search_examples",
```

Comment:
Is the difference in total number of paths across these .i18nrc.json files due to adding some that were not added to the original ones?

Reply:
Yes.

Reply:
We have https://github.com/opensearch-project/dashboards-i18n/blob/main/DEVELOPER_GUIDE.md and https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/packages/osd-i18n. Some day, I will merge those into one and add my future vision into it.